===================================================================
RCS file: /var/lib/cvsd/storage/cvsroot/emacs-w3m/w3mhack.el,v
retrieving revision 1.140
retrieving revision 1.141
diff -u -r1.140 -r1.141
--- emacs-w3m/w3mhack.el	2004/12/23 23:21:03	1.140
+++ emacs-w3m/w3mhack.el	2005/02/10 01:18:44	1.141
@@ -293,9 +293,10 @@
 				   "w3m-xmas.el")
 				  ((boundp 'MULE)
 				   "w3m-om.el")
-				  ((>= emacs-major-version 22)
+				  ((and (>= emacs-major-version 22)
+					(fboundp 'define-coding-system))
 				   '("w3m-e22.el" "w3m-fsf.el"))
-				  ((= emacs-major-version 21)
+				  ((>= emacs-major-version 21)
 				   '("w3m-e21.el" "w3m-fsf.el"))
 				  ((= emacs-major-version 20)
 				   '("w3m-e20.el" "w3m-fsf.el"))
===================================================================
RCS file: /var/lib/cvsd/storage/cvsroot/emacs-w3m/w3m.el,v
retrieving revision 1.1096
retrieving revision 1.1097
diff -u -r1.1096 -r1.1097
--- emacs-w3m/w3m.el	2005/02/02 10:32:30	1.1096
+++ emacs-w3m/w3m.el	2005/02/10 01:18:44	1.1097
@@ -87,9 +87,10 @@
   (cond
    ((featurep 'xemacs)
     (require 'w3m-xmas))
-   ((>= emacs-major-version 22)
+   ((and (>= emacs-major-version 22)
+	 (fboundp 'define-coding-system))
     (require 'w3m-e22))
-   ((= emacs-major-version 21)
+   ((>= emacs-major-version 21)
     (require 'w3m-e21))
    ((= emacs-major-version 20)
     (require 'w3m-e20))
